packages/web/src/components/molecules/Currency/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from 'react'
+import BigNumber from 'bignumber.js'
+import { useGetDevPrice } from 'src/fixtures/uniswap/hooks'
+import SettingContext from 'src/context/settingContext'
+
+interface Props {
+  value?: BigNumber
+}
+
+export const Currency = ({ value }: Props) => {
+  const { data: devPrice } = useGetDevPrice()
+  const { isCurrencyDEV } = useContext(SettingContext)
+  const currencyUnit = isCurrencyDEV ? 'DEV' : 'USD'
+  const currencyValue = isCurrencyDEV ? value : value?.multipliedBy(devPrice)
+
+  return (
+    <span>
+      {currencyValue ? currencyValue.dp(1).toFormat() : '0'} {currencyUnit}
+    </span>
+  )
+}
```

Comment:
@Cold-A-Muse Currency switching is now done by hooks instead of components. Therefore, this component is unnecessary.

See below for the implementation of hooks.

https://github.com/dev-protocol/stakes.social/blob/a4d5c52382d5a15c9948da54a193dd19f82020da/packages/web/src/fixtures/dev-kit/hooks.ts#L42-L51

FYI: The major hooks includes `useCurrency` hooks and returns ` currency`.

`useCurrency().toCurrency` always converts the DEV amount to the currency of your choice (DEV or USD).

Reply:
remove this component in c2801be295587be5ecd094097f1b6c4d0e94feb6.
I think the cause is merge conflict.